src/internal/base-component/index.ts

```diff
@@ -3,24 +3,9 @@
 
 // these styles needed to be imported for every public component
 import './styles.css.js';
-import { PACKAGE_VERSION } from '../environment';
+import { initAwsUiVersions } from '../metrics';
```

Comment:
Just in case, this object is not really used in any metrics, so putting it into that folder is misleading

Reply:
Makes sense, I'll create a separate folder for that.